### DISCUSSION TITLE
Hide Hue scenes from UI?

### DISCUSSION THREAD

**Discussion by FSund** [home-assistant/frontend#12072] [Automations / scripts / scenes]
Each room and zone in the Hue app gets 10 auto-generated scenes, which adds up to a lot of scenes. They all show up under the Scenes tab in HA, which makes it a bit cluttered. Is there any way to hide the scenes from the UI?

**Comment by emufan** [comment]
Just disable them. In Entities list.

But in general, I would like to have an option to disable the generation at all. Unwanted scenes, unwanted entities, unwanted disabled entities. Blowing the registry and make it harder and harder to find, if you are searching something.
